lib/Basic/ObjectFile.cpp

```diff
@@ -0,0 +1,21 @@
+//===--- ObjectFile.cpp - Object File Related Information ------*- C++ -*-===//
```

Comment:
FYI, the `-*- C++ -*-` marker only makes sense in a `.h` file, where it's ambiguous whether it's a C or a C++ header.